rails/db/migrate/20230315111527_drop_legacy_tables.rb

```diff
@@ -0,0 +1,13 @@
+class DropLegacyTables < ActiveRecord::Migration[6.1]
+  def up
+    drop_table :learner_processing_events
+    drop_table :portal_learner_activity_feedbacks
+    drop_table :portal_offering_activity_feedbacks
+    drop_table :portal_offering_embeddable_metadata
+    drop_table :dataservice_blobs
```

Comment:
I might have missed it, but I thought you'd be dropping the "report learner" table. I don't remember its actual name, but it had a single string field that contained all of the learners answers stored in this one field so the old portal details reports could just take this field and join it with the other learners' fields. 

Maybe we already removed that field, but if not, it probably takes up a significant portion of the space in the portal database.